### ISSUE TITLE
Please make button same size

### ISSUE THREAD

**Issue by sonit1810** [divppc/Physio-App#77]
Hi Ihor,

Please see design

https://app.zeplin.io/project/5911792e3ad67ec053171df6/screen/59ca0f5a2f38474ff73aecda

please make "View" button same size with "Create New+" button (keep color, View is blue and Create new is  green , correct color now)

![image](https://user-images.githubusercontent.com/8002065/31600471-cffade1c-b280-11e7-92a7-df8e36cc65b5.png)

**Comment by divppc** [comment]
Hi Sonit, please tell me class of this button, or show it on the page. Can't find it here http://physio.div.agency/invite-new.html

**Comment by sonit1810** [comment]
Hi Ihor,

It's merged from version of invite and invite new too.

Please login to mpa.jsoft.com.au

Then type "t" in searchbox, you will see that list. I added some style to make button the same, but maybe you need fix it a bit for better view, then please give me css,  i will add it to custom.css of site

Thanks!

**Comment by divppc** [comment]
Yes, you did everything correct when changed his paddings. They can't be exactly same, because words are different. One way to make them same, is to set them fixed width and height, instead paddings. But it's bad solution, so i recommend to leave paddings.

**Comment by sonit1810** [comment]
ok Ihor, then leave it, i think it's ok now.
